<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationsConfig</class>
 <widget class="QWidget" name="NotificationsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>424</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="rootLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="kcfg_OsdEnabled">
        <property name="toolTip">
         <string>Check to enable the On-Screen-Display. The OSD briefly displays track data when a new track is played.</string>
        </property>
        <property name="whatsThis">
         <string>Check to enable the On-Screen-Display. The OSD briefly displays track data when a new track is played.</string>
        </property>
        <property name="title">
         <string>Use &amp;On-Screen-Display</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="textLabel1">
             <property name="text">
              <string>&amp;Duration:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="buddy">
              <cstring>kcfg_OsdDuration</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="kcfg_OsdDuration">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>The time in milliseconds for which to show the OSD. The value must be between 500 ms and 10000 ms.</string>
             </property>
             <property name="whatsThis">
              <string>The time in milliseconds for which to show the OSD. The value must be between 500 ms and 10000 ms.</string>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="specialValueText">
              <string>Forever</string>
             </property>
             <property name="suffix">
              <string> ms</string>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>600000</number>
             </property>
             <property name="singleStep">
              <number>1000</number>
             </property>
             <property name="value">
              <number>5000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="textLabel2_3">
             <property name="text">
              <string>Sc&amp;reen:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="buddy">
              <cstring>kcfg_OsdScreen</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KComboBox" name="kcfg_OsdScreen">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>The screen that should display the OSD.</string>
             </property>
             <property name="whatsThis">
              <string>The screen that should display the OSD.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QGroupBox" name="kcfg_OsdUseCustomColors">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Check to enable custom colors for the On-Screen-Display.</string>
           </property>
           <property name="whatsThis">
            <string>Check to enable custom colors for the On-Screen-Display.</string>
           </property>
           <property name="title">
            <string>Use &amp;custom colors</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="textLabel2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>59</width>
                <height>0</height>
               </size>
              </property>
              <property name="whatsThis">
               <string>The color of the OSD text.</string>
              </property>
              <property name="text">
               <string>Text color:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="kcfg_OsdTextColor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>72</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Click to select the color of the OSD text.</string>
              </property>
              <property name="whatsThis">
               <string>The color of the OSD text.</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="color">
               <color>
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="kcfg_OsdUseTranslucency">
           <property name="toolTip">
            <string>If checked, enables translucency if supported by your desktop system.</string>
           </property>
           <property name="text">
            <string>&amp;Translucent</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Font scale:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="kcfg_OsdFontScaling">
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>115</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="kcfg_OsdHideOnFullscreen">
           <property name="toolTip">
            <string>Do not show the On-Screen-Display if another application is focused and running in fullscreen mode.</string>
           </property>
           <property name="text">
            <string>Don't show when a fullscreen application is active</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="kcfg_KNotifyEnabled">
        <property name="toolTip">
         <string>Check to use the system notification.</string>
        </property>
        <property name="text">
         <string>Use &amp;system notifications</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">widgets/Osd.h</include>
  <include location="local">qspinbox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
